pkgs/development/python-modules/asyncinotify/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitLab
+, python
+}:
+
+buildPythonPackage rec {
+  pname = "asyncinotify";
+  version = "4.0.1";
+  format = "flit";
+
+  src = fetchFromGitLab {
+    owner = "Taywee";
+    repo = "asyncinotify";
+    rev = "v${version}";
+    hash = "sha256-DMRuj16KjO+0uAB33UCVPdUiQGzri1b/z9KVqQYp2Po=";
+  };
+
+  checkPhase = ''
+    ${python.interpreter} ${src}/test.py
```

Missing pythonImportsCheck

```suggestion
    ${python.pythonForBuild.interpreter} ${src}/test.py
```

---

Applies and import check added.